strip-path/stripPath.js

```diff
@@ -1,6 +1,16 @@
+import path from 'path';
 
-function stripPath(...args) {
-  return args;
+function stripPath(fullPath, subPath) {
+  if (!subPath) {
+    return fullPath;
+  }
+  const normalizedPath = path.normalize(fullPath);
+  const normalizedSubPath = path.normalize(subPath);
+  let replaced = normalizedPath.replace(normalizedSubPath, '');
```

Comment:
`strippedPath` might be a better name compared to `replaced`

Reply:
Yeah, that's true! 👍